src/Components/SuperAdmins/Form/super-admins-form.module.css

```diff
@@ -0,0 +1,25 @@
+.tableSA {
+  width: 70vh;
+  background-color: #00a6ff;
+}
+button {
```

Comment:
Let's change this selector for a class. This for not having problems with another universal selectors. Remember to add the class in the html tag in your component